mmdet/core/bbox/coder/bucketing_bbox_coder.py

```diff
@@ -26,20 +26,24 @@ class BucketingBBoxCoder(BaseBBoxCoder):
              To avoid too large offset displacements. Defaults to 1.0.
         cls_ignore_neighbor (bool): Ignore second nearest bucket or Not.
              Defaults to True.
+        clip_border (bool): Whether clip the objects outside the border of
```

Comment:
May update the docstring of `clip_border` as below and also change other places.
```python
clip_border (bool, optional): xxxxxx. Defaults to True.
```